sirepo/template/raydata.py

```diff
@@ -78,7 +78,28 @@ def _paths():
 
 
 def background_percent_complete(report, run_dir, is_running):
-    return PKDict(percentComplete=0 if is_running else 100)
+    r = PKDict(percentComplete=0 if is_running else 100)
+    if report != 'pollBlueskyForScansAnimation':
+        return r
+    try:
+        t = float(pkio.read_text(_BLUESKY_POLL_TIME_FILE).strip())
```

Comment:
run_dir.join(_BLUESKY_POLL_TIME_FILE)? I don't know what directory this is in.

Reply:
f5d600154